spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require './lib/facility_factory'
+require './lib/dmv_data_service'
+
+
+RSpec.describe FacilityFactory do
+  it 'exists' do
+    office = FacilityFactory.new 
+  
+    expect(office).to be_an_instance_of(FacilityFactory)
+  end
+
+  describe 'oregon_offices' do
+    it 'has oregon office info' do
+      office = FacilityFactory.new
+      or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
+      office = office.checks_location(or_dmv_office_locations)
```

Comment:
Be careful with your variable naming. You have `office` on line 15 and are using it again here on line 17. To make it easier on yourself you could just do 
```ruby
expect(office.checks_locations(or_dmv_office_locations)).to be_an(Array)
```
Also is this method really checking the locations or is it creating new `Facility` objects??